wagtail_bucketav/tests/test_utils.py

```diff
@@ -1,24 +1,13 @@
 import pytest
-import wagtail_factories
-from wagtail.documents import get_document_model_string
 
 from wagtail_bucketav import utils
 
 
 @pytest.mark.django_db
-def test_get_image_with_key():
-    image = wagtail_factories.ImageFactory()
-    file_key = str(image.file)
+def test_get_document_with_key(document_model):
```

Comment:
We can just inject the test models via Pytest's dependency injection machinery :slightly_smiling_face: